UnitTests/GitCommands.Tests/Helpers/PathUtilTest.cs

```diff
@@ -185,6 +185,7 @@ public void Resolve(string path, string expected)
         }
 
         [TestCase(@"C:\WORK\", @"GitExtensions\", @"C:\WORK\GitExtensions\")]
+        [TestCase(@"C:\WORK\", @" file.txt", @"C:\WORK\ file.txt")]
```

Comment:
Can you please add a testcase for trailing space as well as space before/after /?
(could be in this string)
I
```suggestion
        [TestCase(@"C:\WORK\", @" file.txt ", @"C:\WORK\ file.txt ")]
        [TestCase(@"C:\WORK\", @" \ dir \ file.txt ", @"C:\WORK\ \ dir \ file.txt ")]
```

Reply:
Changed it to `@" file .txt "`. From my limited testing it seems that a space at the end of a file name is not possible in Windows but is possible under Linux.